tests/test_operations.py

```diff
@@ -195,6 +195,18 @@ def test_prompt_with_default():
     eq_(sys.stdout.getvalue(), "%s [%s] " % (s, d))
 
 
+def test_prefix_env_vars_non_strings():
+    """
+    _prefix_env_vars() test int, float, bool as env variable
+    """
+    values = [123, u'123', 1.11, True, False]
+    expect = ['123', '123', '1.11', 'True', 'False']
+    for index, value in enumerate(values):
+        env.shell_env = dict(VAR=value)
```

Comment:
This `env.shell_env` is affecting other tests.
I think using the settings context manager might work: http://docs.fabfile.org/en/1.13/api/core/context_managers.html#fabric.context_managers.settings